### ISSUE TITLE
Bitcoin Dockerfile fails

### ISSUE THREAD

**Issue by vrx00** [serai-dex/serai#230]
When I try to build Bitcoin Profile with 
`docker compose --profile bitcoin up --force-recreate --build`

it fails with following log

```
[serai-dev-bitcoin builder 6/9] RUN gpg -v --keyserver hkp://keys.openpgp.org:80 --recv-keys 4DAF18FE948E7A965B30F9457E296D555E7F63A7 28E72909F1717FE9607754F8A7BEB2621678D37D 74E2DEF5D77260B98BC19438099BAD163C70FBFA 71A3B16735405025D447E8F274810B012346C9A6 E463A93F5F3117EEDE6C7316BD02942421F4889F 9D3CC86A72F8494342EA5FD10A41BDC3F4FAFF1C 287AE4CA1187C68C08B49CB2D11BD4F33F1DB499 F9A8737BF4FF5C89C903DF31DD78544CF91B1514:
#0 0.275 gpg: no running Dirmngr - starting '/usr/bin/dirmngr'
#0 0.275 gpg: waiting for the dirmngr to come up ... (5s)
#0 0.295 gpg: connection to dirmngr established
#0 10.61 gpg: data source: http://keys.openpgp.org:80
#0 10.61 gpg: armor header: Comment: F9A8 737B F4FF 5C89 C903  DF31 DD78 544C F91B 1514
#0 10.61 gpg: armor header: Comment: Jonas Ott (jonas-ott) <git@jonas-ott.de>    
#0 10.61 gpg: armor header: Comment: 287A E4CA 1187 C68C 08B4  9CB2 D11B D4F3 3F1D B499
#0 10.61 gpg: armor header: Comment: jackielove4u <jackielove4u@hotmail.com>     
#0 10.61 gpg: pub  rsa4096/DD78544CF91B1514 2022-04-24  Jonas Ott (jonas-ott) <git@jonas-ott.de>
#0 10.62 gpg: using pgp trust model
#0 10.62 gpg: key DD78544CF91B1514: public key "Jonas Ott (jonas-ott) <git@jonas-ott.de>" imported
#0 10.62 gpg: no running gpg-agent - starting '/usr/bin/gpg-agent'
#0 10.62 gpg: waiting for the agent to come up ... (5s)
#0 10.62 gpg: connection to agent established
#0 10.63 gpg: key 74810B012346C9A6: public key "Wladimir J. van der Laan <laanwj@protonmail.com>" imported
#0 10.63 gpg: pub  rsa4096/099BAD163C70FBFA 2018-02-27  Will Clark <will8clark@gmail.com>
#0 10.63 gpg: key 099BAD163C70FBFA: public key "Will Clark <will8clark@gmail.com>" imported
#0 10.63 gpg: pub  rsa3072/A7BEB2621678D37D 2021-02-18  vertion <vertion@protonmail.com>
#0 10.63 gpg: key A7BEB2621678D37D: public key "vertion <vertion@protonmail.com>" imported
#0 10.63 gpg: pub  rsa4096/7E296D555E7F63A7 2015-07-01  Sjors Provoost <sjors@purpledunes.com>
#0 10.63 gpg: key 7E296D555E7F63A7: public key "Sjors Provoost <sjors@purpledunes.com>" imported
#0 10.63 gpg: Total number processed: 8
#0 10.63 gpg:               imported: 7
------
failed to solve: executor failed running [/bin/sh -c gpg -v --keyserver hkp://keys.openpgp.org:80 --recv-keys ${KEYS2}]: exit code: 2
```

**Comment by TheArchitect108** [comment]
Nice, our build validation caught this. E463A93F5F3117EEDE6C7316BD02942421F4889F was compromised. We can remove this key and builds should work again. https://twitter.com/LukeDashjr/status/1609613748364509184 cc @kayabaNerve

**Comment by vrx00** [comment]
I deleted the key and created the pull request.
https://github.com/serai-dex/serai/pull/231

**Comment by kayabaNerve** [comment]
Closed by #231
